### PR TITLE
Add tanker info menus for players

### PR DESCRIPTION
**Pull request by adam-sandor**
Add special tanker info menus only for players when they slot in. The menu show all important info about the tanker.

I'm aware that I'm adding the same menu multiple times for the same group but I think that's ok because MOOSE has menu deduplication logic. I tried calling the method multiple times and only a single menu appears for the player.

### CODE REVIEW

File: DCAF.Core.lua [hunk 1]
```diff
@@ -5790,6 +5790,12 @@ DCAF.Tanker = {
     TrackSpeed = nil,         -- #number; knots 
     Track = nil, 
     Events = {},              -- dictionary; key = name of event (eg. 'OnFuelState'), value = event arguments 
+    MaxFuelLbs = {
+        ["KC130"] = 66139,
+        ["KC135"] = 199959,
```
Comment (jonasrembratt) [inline DCAF.Core.lua]: I got an error: Unknown tanker type: KC-135

File: DCAF.Core.lua [hunk 2]
```diff
@@ -7899,13 +7905,59 @@ local function buildTankerMenus(caption, scope)
...
+
+ --- Build AAR menus for each player group when player enters plane. Recommended to use with
+  -- default parameters.
+  -- @param #string caption The title of the menu. Default is "AAR"
+  -- @param scope Blue coalition by default for controller menus.
+function DCAF.TankerTracks:BuildF10Menus(caption, scope)
+    buildControllerTankerMenus(caption or "AAR", scope or Coalition.Blue)
+
+    MissionEvents:OnPlayerEnteredAirplane(function(event)
+        buildPlayerTankerMenus(event["IniUnitName"])
```
Comment (jonasrembratt) [inline DCAF.Core.lua]: I only see a call to the `buildPlayerTankerMenus` from here. Won't that mean that each pilot gets one snapshot of the tarnkers' state, as it were the second he slotted in, and then there will be no updates as tankers consume fuel, more tankers gets activated, RTBs, and so on?

**Review by jonasrembratt — COMMENTED** [review]
Two issues:
- There seem to be a problem with how you identify the tankers, to get their "max fuel lbs"
- I can't see how the state gets continously updated after the pilot slots into the plane. Don't you need to schedule an updater, to be run once a minute or something like that?

File: DCAF.Core.lua (hunk 1 above)
Comment (adam-sandor) [reply DCAF.Core.lua]: That's really weird as I did test with different tankers. Can you share the miz?

**Comment by adam-sandor** [comment]
Changed the logic to create a single "Tanker info" menu item for each player that will display all active tanker states.

**Pull request merged**